packages/mdc-textfield/adapter.js

```diff
@@ -28,16 +28,16 @@
 let NativeInputType;
 
 /**
- * Adapter for MDC Textfield.
+ * Adapter for MDC TextField.
```

Comment:
If other adapters are any indication, we should use "Text Field" here and L34 (like "Form Field" and "Simple Menu" in their respective adapters).